app/Repository/Supply/NetworkCampaignRepository.php

```diff
@@ -80,14 +88,14 @@ public function save(Campaign $campaign): void
         /** @var Banner $domainBanner */
         foreach ($banners as $domainBanner) {
             $banner = $domainBanner->toArray();
-            $banner['uuid'] = $banner['id'];
-            $banner['serve_url'] = ForceUrlProtocol::change($banner['serve_url']);
-            $banner['click_url'] = ForceUrlProtocol::change($banner['click_url']);
-            $banner['view_url'] = ForceUrlProtocol::change($banner['view_url']);
+            $banner[self::BANNER_UUID_FIELD] = $banner['id'];
```

Comment:
The 'id' has it's const. It should be used here
```suggestion
            $banner[self::BANNER_UUID_FIELD] = $banner[self::BANNER_ID_FIELD];
```